librz/bin/format/pe/pe.h

```diff
@@ -69,6 +69,17 @@
 
 #define PE_SCN_ALIGN_MASK 0x00F00000
 
+typedef struct rz_bin_pe_reloc_block_t {
+	ut32 page_rva;
+	ut32 block_size;
+} RzBinPeRelocBlock;
+
+// encoding is 12 bits for type and 4 bits for offset. 
+// See: https://learn.microsoft.com/en-us/windows/win32/debug/pe-format#the-reloc-section-image-only
+#define PE_RELOC_ENT_TYPE(val)   ((val) >> 12)
+#define PE_RELOC_ENT_OFFSET(val) ((val)&0b111111111111)
```

Comment:
i'm not sure how `0b111111111111` is portable. use hex (`0xfff`).

Reply:
You're right, totally forgot about that. I'll fix it